test2.js

```
@@ -0,0 +1,40 @@
+}
+
+/* eslint-disable no-alert, no-console */
+
+function badFunction() {
+  alert('This is an alert!'); 
+  console.log('Logging to the console.'); 
+  let x = 10;
+  if (x = 20) {
+    console.log('x is now 20');
+  }
+  for (let i = 0; i < 10; i--) { 
```

Biome

[error] The update clause in this loop moves the variable in the wrong direction.



(lint/correctness/useValidForDirection)